### ISSUE TITLE
Nightly GitHub Action failure

### ISSUE THREAD

**Issue by robert-scheck** [rpki-client/rpki-client-portable#71]
Our nightly GitHub Action for the container image [fails](https://github.com/rpki-client/rpki-client-container/runs/8127814776?check_suite_focus=true#step:9:1919) since today with:

```
#34 109.8 /tmp/rpki-client-portable/src/parser.c:497: undefined reference to `aspa_parse'
#34 109.8 /usr/lib/gcc/x86_64-alpine-linux-musl/11.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: rpki_client-parser.o: in function `parse_entity':
#34 109.8 /tmp/rpki-client-portable/src/parser.c:647: undefined reference to `aspa_buffer'
#34 109.8 /usr/lib/gcc/x86_64-alpine-linux-musl/11.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: /tmp/rpki-client-portable/src/parser.c:648: undefined reference to `aspa_free'
#34 109.8 /usr/lib/gcc/x86_64-alpine-linux-musl/11.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: rpki_client-parser.o: in function `proc_parser_aspa':
#34 109.8 /tmp/rpki-client-portable/src/parser.c:505: undefined reference to `aspa_free'
#34 109.8 collect2: error: ld returned 1 exit status
```

**Comment by cjeker** [comment]
Fixed in f8e526b2064929ebf8b6529e2ae3f913abcc4a87

**Comment by robert-scheck** [comment]
Confirming [successful run](https://github.com/rpki-client/rpki-client-container/runs/8135590548) with commit https://github.com/rpki-client/rpki-client-portable/commit/f8e526b2064929ebf8b6529e2ae3f913abcc4a87 - thank you.
